easybuild/easyconfigs/r/RMBlast/RMBlast-2.2.28.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Binary'
+
+name = 'RMBlast'
+version = '2.2.28'
+
+homepage = 'http://www.repeatmasker.org/RMBlast.html'
+description = """a RepeatMasker compatible version of NCBI BLAST"""
+
+toolchain = {'name': 'dummy', 'version': ''}
```

Comment:
@pescobar Any particular reason you're opting for a binary install here, especially since we already have support for building `RMBlast` from source?